src/redux/actions/syncBattle.ts

```diff
@@ -117,6 +117,9 @@ export const syncBattle = createAsyncThunk<CalcdexBattleState, SyncBattlePayload
       return;
     }
 
+    if (battleState.format.endsWith('bo3')) {
+      battleState.format = battleState.format.slice(0, -3);
+    }
```

Comment:
ah I see Showdown added this, which is cool! would definitely break things down the line, so good looks c: